docs/reference/operators/sort.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Sort
+---
+
+The Sort operator sorts the output. This operator is only enabled for data
+sources of type array.
```

Comment:
Does it operate on the client or the server?

Reply:
Only on the server AFAIU. cc @flyth and @blanquicet

Reply:
I added a TODO to document where all operators are used.